subprojects/docs/src/docs/userguide/jvm/java_testing.adoc

```diff
@@ -282,6 +282,10 @@ include::sample[dir="snippets/testing/junit-xml-results/kotlin",files="build.gra
 include::sample[dir="snippets/testing/junit-xml-results/groovy",files="build.gradle[tags=configure-content]"]
 ====
 
+===== omitSystemOutLog & omitSystemErrLog
+
+Allows configuring XML exporter to omit the system error and system out logs in the result file.
```

Comment:
To match the documentation should have the flag in the text and the title. It would also be nice to have a sentence about why this would be useful. When would someone want to change the option to true?

```suggestion
When enabled, the `omitSystemErrLog` option allows configuring XML exporter to omit the system error in the result file. 
While the `omitSystemErrLog` option allows configuring XML exporter to omit the system out logs in the result file.
```